docs/JSON-RPC-API/Authentication.md

```diff
@@ -0,0 +1,88 @@
+    permissions=["net:*","eth:blockNumber"]
+    
+    [Users.username2]
+    password = "$2b$10$6sHt1J0MVUGIoNKvJiK33uaZzUwNmMmJlaVLkIwinkPiS1UBnAnF2"
+    permissions=["net:version","admin:*"]
+    ```
+
+Each user requiring JSON-RPC access is listed with: 
+
+* Hash of the user password. Use the [`password`](../Reference/Pantheon-CLI-Syntax.md#password) subcommand to generate the hash. 
+* JSON-RPC permissions. 
+
+!!! example "password Subcommand"
+    ```bash
+    pantheon password hash --password=pegasys
+    ```
+    
+## JSON-RPC Permissions 
+
+Each user has a list of permissions strings defining the methods they can access. To give access to: 
+
+* All API methods, specify `["*:*"]`.
+* All API methods in an API group, specify `["<api_group>:*"]`. For example, `["eth:*"]`. 
+* Specific API methods, specify `["<api_group>:<method_name>"]`. For example, `["admin:peers"]`.
+
+To explicitly specify a user cannot access any methods, include the user with an empty permissions list (`[]`). 
+Users with an empty permissions list and users not included in the credentials file cannot access any JSON-RPC
+methods. 
+    
+## Enabling Authentication 
+ 
+Use the [` --rpc-http-authentication-enabled`](../Reference/Pantheon-CLI-Syntax.md#rpc-http-authentication-enabled) or 
+ [`--rpc-ws-authentication-enabled`](../Reference/Pantheon-CLI-Syntax.md#rpc-ws-authentication-enabled)
+ options to require authentication for the JSON-RPC API.
+  
+Use the [`--rpc-http-authentication-credentials-file`](../Reference/Pantheon-CLI-Syntax.md#rpc-http-authentication-credentials-file)
+and [`--rpc-ws-authentication-credentials-file`](../Reference/Pantheon-CLI-Syntax.md#rpc-ws-authentication-credentials-file) 
+options to specify the [credentials file](#credentials-file).  
+
+## Obtaining an Authentication Token 
+
+To obtain an authentication token, make a request to the `/login` endpoint with your username and password. 
+
+!!! example
+    ```bash tab="curl HTTPS request"
+    $ curl -X POST --data '{"username":"username1","password":"pegasys"}' <JSON-RPC-https-endpoint:port>/login
+    ```
+    
+    ```json tab="JSON result"
+    {"token":"eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJwZXJtaXNzaW9ucyI6WyIqOioiXSwidXNlcm5hbWUiOiJ1c2VyMiIsImlhdCI6MTU1MDQ2MDYwNCwiZXhwIjoxNTUwNDYwOTA0fQ.l2Ycqzl_AyvReXBeUSayOlOMS_E8-DCuz3q0Db0DKD7mqyl6q-giWoEtfdWzUEvZbRRi2_ecKO3N6JkXq7zMKQAJbVAEzobfbaaXWcQEpHOjtnK4_Yz-UPyKiXtu7HGdcdl5Tfx3dKoksbqkBl3U3vFWxzmFnuu3dAISfVJYUNA"}
+    ``` 
+
+Authentication tokens expire 5 minutes after being generated. It is necessary to generate a new authentication 
+token if access is required after token expiration.     
+    
+## Using an Authentication Token to Make Requests 
+
+Specify the authentication token as a `bearer` token in the JSON-RPC request header. 
+
+If using Postman, specify a key:value pair in the _Headers_ tab where the key is `bearer` and the value is the token 
```

Comment:
I think the key is case sensitive - should be `Bearer`

Reply:
Will be covered by updates for PR#891